patches/chrome-android-java-src-org-chromium-chrome-browser-toolbar-ToolbarManager.java.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+index 5e062152c274c9f03aecd657ac9009705ac48db5..c1fbcd05150187179249c901e164aa4c1fa78645 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/toolbar/ToolbarManager.java
+@@ -777,13 +777,7 @@ public class ToolbarManager implements ScrimObserver, ToolbarTabController, UrlF
+         };
+ 
+         final OnClickListener shareButtonListener = v -> {
+-            recordBottomToolbarUseForIPH();
+-            RecordUserAction.record("MobileBottomToolbarShareButton");
+-            boolean isIncognito = false;
+-            if (mTabModelSelector != null) {
+-                isIncognito = mTabModelSelector.getCurrentTab().isIncognito();
+-            }
+-            mActivity.onShareMenuItemSelected(false, isIncognito);
++            mActivity.addOrEditBookmark(mActivity.getActivityTab());
```

Comment:
I'm confused about this change, why is the share button being used for add/edit bookmark?

Reply:
ok, I see what's happening now. Same as below can we do 
`mActivity.addOrEditBookmark(mActivity.getActivityTab()); return;` as the first line in this method and leave the rest?

Reply:
done as the above via /**/